### PR TITLE
feat: add sortedSetLengthByScore

### PR DESCRIPTION
**Pull request by nand4011**
Add the sorted set length by score API, integration tests, and documentation.

### CODE REVIEW

File: src/Cache/CacheOperationTypes/CacheOperationTypes.php [hunk 1]
```diff
@@ -3239,7 +3239,83 @@ class SetRemoveElementError extends SetRemoveElementResponse
...
+ */
+class SortedSetLengthByScoreSuccess extends SortedSetLengthByScoreResponse
+{
+    private int $length;
+
+    public function __construct(_SortedSetLengthByScoreResponse $response)
+    {
+        parent::__construct();
+        $this->length = $response->getFound() ? $response->getFound()->getLength() : 0;
+    }
```
Comment (malandis) [inline src/Cache/CacheOperationTypes/CacheOperationTypes.php:3296]: Just leaving a note: the "or 0" case should never happen. I opted to keep the protobuf responses out of here and pass concrete values to here from the data client.

File: src/Cache/CacheOperationTypes/CacheOperationTypes.php [hunk 2]
```diff
@@ -3239,7 +3239,83 @@ class SetRemoveElementError extends SetRemoveElementResponse
...
+     * @return SortedSetLengthByScoreError|null Returns the error subtype if the request returned an error and null otherwise.
+     */
+    public function asError(): ?SortedSetLengthByScoreError
+    {
+        if ($this->isError()) {
+            return $this;
+        }
+        return null;
+    }
+}
```
Comment (malandis) [inline src/Cache/CacheOperationTypes/CacheOperationTypes.php:3283]: Length is the hit/miss/error pattern

**Review by malandis — COMMENTED** [review]
In the JS SDK for the collections, length is hit/miss/error

File: src/Cache/CacheOperationTypes/CacheOperationTypes.php (hunk 2 above)
Comment (nand4011) [reply src/Cache/CacheOperationTypes/CacheOperationTypes.php]: Updated.

**Review by malandis — APPROVED** [review]

**Pull request merged**